src/zui/ZUIPersonGridEditCell.tsx

```diff
@@ -303,6 +305,23 @@ const ZUIPersonGridEditCell: FC<{
                             />
                           );
                         })}
+
+                        {autoComplete.inputValue.length >= 3 &&
+                          !personSelect.autoCompleteProps.isLoading &&
+                          searchResults.length === 0 && (
+                            <Button
+                              color="primary"
+                              onClick={onCreate}
+                              startIcon={<PersonAdd />}
+                              sx={{
+                                justifyContent: 'flex-start',
+                                m: 2,
+                              }}
+                              variant="outlined"
+                            >
+                              <Msg id={messageIds.createPerson.createBtn} />
+                            </Button>
+                          )}
```

Comment:
Now that I've played with the UI, I think it would be useful to have this button at the end of the list even if the list is not empty.